admin/src/components/Navbar.jsx

```
@@ -1,20 +1,39 @@
       <img className="w-[max(5%,20px)] " src={logo} alt="" />
-      <button
-        onClick={() => setToken("")}
-        className="bg-gray-600  text-white px-4 py-2 sm:px-7 sm:py-2 rounded-full text-xs sm:text-sm"
-      >
-        Logout
-      </button>
+      <div className="flex items-center gap-2">
+        <button
+          onClick={() => setShowChangePassword(true)}
+          className="bg-gray-100 hover:text-white  text-gray-700 border border-dashed border-zinc-500 px-4 py-2 sm:px-7 sm:py-2 rounded-full text-xs sm:text-sm"
+        >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)